pkg/makefile

```diff
@@ -161,10 +161,10 @@ deb: stage_deb
 		-s dir \
 		-d nfs-kernel-server \
 		-d net-tools \
-		-d libdevmapper \
-		-d libdevmapper-event \
+		-d libdevmapper1.02.1 \
+		-d libdevmapper-event1.02.1 \
```

Comment:
Ubuntu 14.04 doesn't have a "libdevmapper" or "libdevmapper-event" repo, only the versioned ones (with the versions in the name).